exo-run/src/app-runner.ls

```diff
@@ -1,74 +1,44 @@
 require! {
-  'async'
   'chalk': {red}
-  'child_process'
-  './docker-runner' : DockerRunner
   'events' : {EventEmitter}
-  '../../exosphere-shared' : {compile-service-routes, DockerHelper}
-  'nitroglycerin' : N
-  'port-reservation'
+  '../../exosphere-shared' : {DockerHelper}
   'path'
-  './service-runner' : ServiceRunner
-  'wait' : {wait-until}
+  './service-watcher' : ServiceWatcher
 }
 
 
 # Runs the overall application
 class AppRunner extends EventEmitter
 
   ({@app-config, @logger}) ->
+    @env =
+      EXOCOM_PORT: 80 or process.env.EXOCOM_PORT
```

Comment:
I think this should be `process.env.EXOCOM_PORT or 80` otherwise the `80` will always succeed first and be used.